Cloudinary/Classes/Core/Features/Helpers/Results/Helpers/CommonResultKeys.swift

```diff
@@ -25,38 +25,40 @@
 import Foundation
 
 internal enum CommonResultKeys: CustomStringConvertible {
-    case publicId, format, version, resourceType, urlType, createdAt, length, width, height, x, y, url, secureUrl, exif, metadata, faces, colors, tags, moderation, context, phash, info, accessControl, eager, qualityAnalysis, coordinates, accessibilityAnalysis, originalFilename
+    case publicId, format, version, resourceType, urlType, createdAt, length, width, height, x, y, url, secureUrl, exif, metadata, faces, colors, tags, moderation, context, phash, info, accessControl, eager, qualityAnalysis, coordinates, accessibilityAnalysis, originalFilename, assetFolder, displayName
     
     var description: String {
         switch self {
-        case .publicId:              return "public_id"
-        case .format:                return "format"
-        case .version:               return "version"
-        case .resourceType:          return "resource_type"
-        case .urlType:               return "type"
-        case .createdAt:             return "created_at"
-        case .length:                return "bytes"
-        case .width:                 return "width"
-        case .height:                return "height"
-        case .x:                     return "x"
-        case .y:                     return "y"
-        case .url:                   return "url"
-        case .secureUrl:             return "secure_url"
-        case .exif:                  return "exif"
-        case .metadata:              return "image_metadata"
-        case .faces:                 return "faces"
-        case .colors:                return "colors"
-        case .tags:                  return "tags"
-        case .moderation:            return "moderation"
-        case .context:               return "context"
-        case .phash:                 return "phash"
-        case .info:                  return "info"
-        case .eager:                 return "eager"
-        case .accessControl:         return "access_control"
-        case .qualityAnalysis:       return "quality_analysis"
-        case .accessibilityAnalysis: return "accessibility_analysis"
-        case .coordinates:           return "coordinates"
-        case .originalFilename:      return "original_filename"
+        case .publicId:                 return "public_id"
```

Comment:
I understand you added more variables to the description variable but the indentation, tabs and spaces should stay the same and they changed.